blog/views.py

```diff
@@ -67,10 +67,8 @@ def article_edit(id):
     if request.method == 'POST':
         article.title = request.form.get('title')
         article.content = request.form.get('content')
-        if bool(request.form.get('published')):
+        if bool(request.form.get('published') and not article.publication_date):
```

Comment:
We actually need to convert to boolean the `request.form.get('published')` part:

```python
if bool(request.form.get('published')) and not article.publication_date:
```